packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/PreviewSell/index.tsx

```diff
@@ -103,159 +229,292 @@ class PreviewSell extends PureComponent<InjectedFormProps<{}, Props> & Props> {
+                                {Currencies[counterCoinTicker].units[counterCoinTicker].symbol}
+                                {saleAmount}
+                              </Value>
+                              <AdditionalText>
+                                {coinToString({
+                                  unit: {
+                                    symbol: coins[account.baseCoin].coinTicker
+                                  },
+                                  value: saleInCoin
+                                })}
+                              </AdditionalText>
+                            </>
+                          )
+                        }
+                      })}
+                    </RowTextWrapper>
+                  </RowText>
+                </RowItem>
+                <RowItem>
+                  <RowItemContainer>
+                    <TopRow>
+                      <RowIcon>
+                        <RowText>
+                          <FormattedMessage
+                            id='copy.coin_network_fee'
+                            defaultMessage='Network Fee'
+                          />
+                        </RowText>
+                        <IconWrapper>
+                          <Icon
+                            name='question-in-circle-filled'
+                            size='16px'
+                            color={this.state.isSetNetworkFee ? 'blue600' : 'grey300'}
+                            onClick={() => this.toggleNetworkFeeToolTip()}
+                          />
+                        </IconWrapper>
+                      </RowIcon>
+                      <RowText data-e2e='sbTransactionFee'>
+                        <RowTextWrapper>
+                          {Currencies[counterCoinTicker].units[counterCoinTicker].symbol}
+                          {formatFiat(feeInFiat)}
+                          <AdditionalText>
+                            {coinToString({
+                              unit: {
+                                symbol: coins[account.baseCoin].coinTicker
+                              },
+                              value: convertBaseToStandard(
+                                account.baseCoin,
+                                this.networkFee(this.props.payment)
+                              )
+                            })}
+                          </AdditionalText>
+                        </RowTextWrapper>
+                      </RowText>
+                    </TopRow>
+                    {this.state.isSetNetworkFee && (
+                      <ToolTipText>
+                        <Text size='12px' weight={500} color='grey600'>
+                          <TextGroup inline>
+                            <Text size='14px'>
+                              <FormattedMessage
+                                id='modals.simplebuy.confirm.network_fees'
+                                defaultMessage='Network fees are set by the Bitcoin network.'
```

Comment:
why would this say "Bitcoin" network?  it's not always going to be Bitcoin right?

Reply:
yeah should be coin specific, added a fix